.circleci/deploy-release.yml

```diff
@@ -33,7 +30,12 @@ jobs:
             if [ "$CURRENT_VERSION" = "$OLD_VERSION" ]; then
               echo "MARKETING_VERSION has not changed. Exiting..."
               circleci-agent step halt
+              exit 0
             fi
+
+            git fetch origin $CIRCLE_BRANCH
```

Comment:
Is this line needed? I guess we already have the up-to-date version locally at this point and checking out (as done on the line below) would be enough, wouldn't it?

Reply:
So happy when I receive such comments 🙏 
Not needed indeed 😄 - thought the same when reviewing the script.
Left there as it was part of the old process. Removed in last commits 😉